webui/react/src/components/ConfigPolicies.tsx

```diff
@@ -161,6 +163,12 @@ const ConfigPoliciesTab: React.FC<TabProps> = ({ workspaceId, global, type }: Ta
     setDisabled(hasErrors(form) || form.getFieldValue(YAML_FORM_ITEM_NAME) === initialYAML);
   };
 
+  const docsLink = (
+    <Link external path={paths.docs('/manage/config-policies.html')} popout>
+      Learn more
+    </Link>
+  );
+
```

Comment:
this PR is merge blocked on https://github.com/determined-ai/determined/pull/9969, which will add this page to the docs

Reply:
the final URL will be this:
https://docs.determined.ai/latest/manage/config-policies.html